packages/react-liveness/src/components/FaceLivenessDetector/service/types/credentials.ts

```diff
@@ -0,0 +1,15 @@
+export interface AwsCredentials {
```

Comment:
These doesn't feel right. Are we sure we are covering all our bases here in copying the types? I don't see any issues in depending on the aws-sdk types tbh.

Reply:
@thaddmt Is this just directly copied from the AWS SDK types?